optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticUtils.java

```diff
@@ -68,8 +69,12 @@ public static String getStandardDeviationString(double[] standardDeviationDouble
         StringBuilder standardDeviationString = new StringBuilder(standardDeviationDoubles.length * 9);
         // Abbreviate to 2 decimals
         // We don't use a local sensitive DecimalFormat, because other Scores don't use it either (see PLANNER-169)
+        DecimalFormatSymbols decimalFormatSymbols = DecimalFormatSymbols.getInstance();
```

Comment:
+1, will fix ASAP. Also, according to the docs, Locale.US will always be present (or a compatible substitute)